dll/shellext/ntobjshex/foldercommon.h

```diff
@@ -42,7 +42,7 @@ class CFolderViewCB :
         return E_NOTIMPL;
     }
 
-    virtual HRESULT STDMETHODCALLTYPE Initialize(IShellView* psv)
+    virtual STDMETHODIMP Initialize(IShellView* psv)
```

Comment:
Declaration should take priority more than implementation. `virtual` keyword will be abstracted.

```suggestion
    STDMETHOD(Initialize)(IShellView* psv)
```